src/types/transaction.ts

```diff
@@ -89,3 +87,11 @@ export type IFinishedTxReceipt = DistributiveOmit<
   timestamp: number;
   blockNumber: number;
 };
+
+export interface IFullTxHistoryValueTransfer {
+  readonly from: TAddress;
+  readonly to: TAddress;
+  readonly asset: Asset;
+  readonly amount?: string; // Hex
+}
+
```

Comment:
Unnecesary newline

Reply:
Files should have a single line at the end of a file. It looks like some files in your PR have two or more lines, which is unnecessary.